test/cryptographic-identities.ts

```diff
@@ -8,16 +8,40 @@ import Future = require("fibers/future");
 import util = require("util");
 import helpers = require("./../lib/helpers");
 import validatorsModule = require("./../lib/validators/cryptographic-identity-validators");
+import logger = require("../lib/common/logger");
 import commandsModule = require("./../lib/commands/cryptographic-identities");
-var assert = require("chai").assert;
-
-function createTestInjector(): IInjector {
-	require("../lib/common/logger");
+import x509 = require("./../lib/x509");
+import assert = require("assert");
+var todaysTime = new Date().getTime(),
```

Comment:
Tomorrow's date is needed in order to trick the x509 module into validating the certificate's expiry date. By getting **tomorrow** as an expiry date we ensure that the certificate will always be valid no matter what.
In a discussion with @rosen-vladimirov we decided against hard-coding the date